.github/workflows/release.yaml

```diff
@@ -0,0 +1,90 @@
+name: "Release ODH"
+on:
+  pull_request:
+    types:
+      - closed
+jobs:
+  gh-release:
+    if: github.event.pull_request.merged && startsWith(github.event.pull_request.title, 'ODH Release') && endsWith(github.event.pull_request.title, 'Version Update')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+      - name: Get release data from pr
+        uses: peter-evans/find-comment@v3
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body-includes: "#Release#"
+      - name: Set version/tracker-url in env
+        run: |
+          echo "VERSION=$(echo "${{ steps.fc.outputs.comment-body }}" | grep version | cut -d "=" -f2)" >> $GITHUB_ENV
```

Comment:
something like ```awk -F= '$1 == "version" {print $2}'``` ?

Reply:
this works 
`awk -F' ' '{print $2}' | awk -F= '$1 == "version"{print $2}'`